app/apollo/resolvers/channel.js

```diff
@@ -309,8 +309,13 @@ const channelResolvers = {
       }
 
       const path = `${org_id.toLowerCase()}-${channel.uuid}-${name}`;
-      const bucketName = conf.storage.getChannelBucket(channel.data_location);
-      const handler = storageFactory(logger).newResourceHandler(path, bucketName, channel.data_location);
+      const { data_location=null } = channel;
+      const bucketConfObj = {
+        type: 'active',
+        region: data_location,
+        kind: 'configs',
+      };
+      const handler = storageFactory(logger).newResourceHandler({ path, bucketConfObj, data_location });
```

Comment:
`data_location` is part of the `bucketConfObj` _and_ a separate arg to `newResourcehandler`.  Should eliminate from one of the two... preferably keep it in the conf obj only unless there's a technical reason it can't be removed from the function signature.
```suggestion
      const handler = storageFactory(logger).newResourceHandler({ path, bucketConfObj });
```